main/src/main/scala/sbt/Keys.scala

```diff
@@ -264,6 +264,7 @@ object Keys {
   val bgRunMain = inputKey[JobHandle]("Start a provided main class as a background job")
   val fgRunMain = inputKey[Unit]("Start a provided main class as a foreground job")
   val bgCopyClasspath = settingKey[Boolean]("Copies classpath on bgRun to prevent conflict.")
+  val layeringStrategy = settingKey[LayeringStrategy]("Creates the classloader layering strategy for the particular configuration.")
```

Comment:
Maybe we should come up with more verbose or intuitive naming for this key and class?

Reply:
I changed it to [cC]lassLoaderLayeringStrategy which is definitely verbose and hopefully more intuitive.